jwql/jwql_monitors/generate_preview_images.py

```diff
@@ -584,6 +584,9 @@ def generate_preview_images(overwrite, programs=None):
     all_programs = [os.path.basename(item) for item in glob.glob(os.path.join(SETTINGS['filesystem'], 'public', 'jw*'))]
     all_programs.extend([os.path.basename(item) for item in glob.glob(os.path.join(SETTINGS['filesystem'], 'proprietary', 'jw*'))])
 
+    # Remove any repeats
+    all_programs = list(set(all_programs))
```

Comment:
Might want to double check on this from #1717

Reply:
Moved the removal of duplicates up here, from where it was after #1717